framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1305,6 +1306,18 @@ protected void annotateInheritedFromClass(/*@Mutable*/ AnnotatedTypeMirror type,
     }
 
     /**
+     * Return an AnnotatedNullType with {@code annosToAdd} added
```

Comment:
I find the `added` confusing, because I looked for the parameter to which the annotations would be added.
Are there any other annotations already present on the null type? Or will those be the only annotations on it?
I would clarify the parameter name and documentation.